benchmarking/blackbox_repository/conversion_scripts/scripts/lcbench/lcbench.py

```diff
@@ -0,0 +1,75 @@
+import zipfile
+import urllib
+import pandas as pd
+import numpy as np
+
+from benchmarking.blackbox_repository.blackbox_tabular import serialize, BlackboxTabular
+import syne_tune.search_space as sp
+
+from benchmarking.blackbox_repository.conversion_scripts.scripts.lcbench.api import Benchmark
+from benchmarking.blackbox_repository.conversion_scripts.utils import repository_path
+from syne_tune.util import catchtime
+
+
+def convert_task(bench, dataset_name):
+    n_config = 2000
+    num_epochs = 52
+    configs = [bench.query(dataset_name=dataset_name, tag="config", config_id=i) for i in range(n_config)]
+    hyperparameters = pd.DataFrame(configs)
+    # remove constant columns
+    hyperparameters = hyperparameters.loc[:, (hyperparameters != hyperparameters.iloc[0]).any()]
+    objective_evaluations = np.zeros((n_config, 1, num_epochs, 2))
+
+    configuration_space = {
+        "num_layers": sp.randint(1, 5),
+        "max_units": sp.lograndint(64, 512),
+        "batch_size": sp.lograndint(16, 512),
+        "learning_rate": sp.loguniform(1e-4, 1e-1),
+        "weight_decay": sp.uniform(1e-5, 1e-1),
+        "momentum": sp.uniform(0.1, 0.99),
```

Comment:
These are infinite ranges. Is there a surrogate involved here somewhere?

Reply:
Blackboxes do not have a surrogate in themselves since they just describe the data. 
That being said, this particular example needs a surrogate unless you restrict the search to a list of evaluated candidates.